src/spring-cloud/azext_spring_cloud/_enterprise.py

```diff
@@ -552,3 +562,119 @@ def _try_stream_build_log(progress_bar, streaming_url, primary_key, build_result
         else:
             return True, "Failed to get build logs with status code '{}' and reason '{}'".format(
                     response.status_code, response.content)
+
+def _create_default_buildpack_binding_for_application_insights(cmd, client, resource_group, name, location,
```

Comment:
Putting these buildpack bindings related things to another file

Reply:
sure.

Reply:
Updated.